### ISSUE TITLE
rotate() map to rotateZ() in WEBGL?

### ISSUE THREAD

**Issue by shiffman** [processing/p5.js#2351]
In Processing,  if `rotate()` is called in `P3D` it is the same as `rotateZ()`. This allows code to move back and forth between 2D and 3D more easily when testing. Advisable to do the same here?

At the moment, if you call `rotate()` in `WEBGL`, the following error appears:

```
p5.min.js:8 Uncaught TypeError: Cannot read property 'transpose3x3' of null
    at d.Matrix.inverseTranspose (VM48 p5.min.js:8)
    at d.RendererGL._setMatrixUniforms (VM48 p5.min.js:9)
    at d.RendererGL.f.RendererGL.drawBuffers (VM48 p5.min.js:9)
    at d.RendererGL.rect (VM48 p5.min.js:9)
    at e.d.rect (VM48 p5.min.js:4)
    at draw (VM116 about:srcdoc:54)
    at e.d.redraw (VM48 p5.min.js:6)
    at e.<anonymous> (VM48 p5.min.js:4)
    at e.<anonymous> (VM48 p5.min.js:4)
    at new e (VM48 p5.min.js:5)
```

**Comment by Spongman** [comment]
i fixed this in #2366, however it doesn't address the issue that the origins are different between P2D & WEBGL modes. obviously it would be a rather large breaking change to move the WEBGL origin to the top-left...

**Comment by lmccart** [comment]
fixed with #2366
